app/controllers/embed_controller.rb

```diff
@@ -54,6 +55,10 @@ def embed_request
     render body: e.to_s, status: :unsupported_media_type
   end
 
+  def set_transcripts_flag_per_param
+    Settings.enabled_features.transcripts = true if params[:transcripts] == 'true'
```

Comment:
This settings object is a global, so you can't change it on a per-request basis.

Reply:
@edsu I don't think that's a viable pattern in Argo either.  Any other request served by the same process will be affected by that setting.